src/element/textElement.ts

```diff
@@ -856,3 +858,26 @@ export const isMeasureTextSupported = () => {
   );
   return width > 0;
 };
+
+/**
+ * Unitless line height
+ *
+ * In previous versions we used `normal` line height, which browsers interpret
+ * differently, for different fonts.
+ *
+ * 1.25 aligns with `normal` for Virgil in WebKit and Blink. Gecko (FF) uses 1.3.
+ * 1.15 aligns with `normal` for `Helvetica` and 1.2 aligns with `normal` for `Cascadia`
+ */
+
+const DEFAULT_LINE_HEIGHT = {
+  [FONT_FAMILY.Virgil]: 1.25 as ExcalidrawTextElement["lineHeight"],
+  [FONT_FAMILY.Helvetica]: 1.15 as ExcalidrawTextElement["lineHeight"],
+  [FONT_FAMILY.Cascadia]: 1.2 as ExcalidrawTextElement["lineHeight"],
+};
```

Comment:
if it's used just in one place doesn't matter. The only argument for why we can keep it here is so that it's not exported and used ad hoc. But it's sad that we can't have all constants in one file so it's easy to find them (and for people not familiar with the codebase to be able to edit them).

Reply:
> The only argument for why we can keep it here is so that it's not exported and used ad hoc.
That's what I mean by it being used in one place. Moving it to constants and exporting it is unnecessary in this case.
As per familiarness, editors are smart enough to point users to the correct place